### ISSUE TITLE
Headless browser is not keeping the history

### ISSUE THREAD

**Issue by Cyb3r-Monk** [cmu-sei/GHOSTS#193]
When I use headless Chrome with the current user's profile (user-data-dir is set to full path of the profile), the browser doesn't keep the browsing history. I don't know if it's a limitation of the Selenium or headless browser itself.  Is it possible to keep the browsing history when using headless?

**Comment by sei-dupdyke** [comment]
Do you have incognito set by chance?

```
if (handler.HandlerArgs.ContainsKeyWithOption("incognito", "true"))
{
   options.AddArguments("--incognito");
}
```

which would look like this in the config:

```
"HandlerArgs": {
        "isheadless": "false",
        ...
        "incognito": "true",
```

**Comment by Cyb3r-Monk** [comment]
No, I don't have the incognito enabled. I can see the browser history when using headful Chrome. My guess is it's related to being headless. Maybe the new headless version doesn't have this problem.

**Comment by sei-dupdyke** [comment]
Hmm, this doesn't appear to be related to ghosts—if I use headless, that history doesn't show in the normal browser, so I assume that is an issue with their handling of headless.
